core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -816,6 +833,10 @@ class SparkContext(config: SparkConf) extends Logging {
   /**
    * Read a text file from HDFS, a local file system (available on all nodes), or any
    * Hadoop-supported file system URI, and return it as an RDD of Strings.
+   *
+   * @param path URI to Hadoop-supported file system
+   * @param minPartitions suggested value of the minimal splitting number for input data
```

Comment:
I think the doc string is hard to understand. "Suggested minimum number of partitions for the resulting RDD"?